cpp/ps0000/solution 1.cpp

```diff
@@ -1,7 +1,9 @@
-#include<iostream>
+#include <iostream> // 2/2
 using namespace std;
-int main ()
-[
-cout <<"Hello,World!/n";
-return 0;
-]
\ No newline at end of file
+
+int main () // 0.5/1
+{
+  cout << "Hello, World!/n"; // 3.5/4
+  return 0; // 1/1
+}
+// great work 😎 9/10 ⭐
```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit